app-text/poppler/poppler-0.63.0.recipe

```diff
@@ -0,0 +1,433 @@
+SUMMARY="A PDF rendering library"
+DESCRIPTION="Poppler is a PDF rendering library based on the xpdf-3.0 code \
+base."
+HOMEPAGE="https://poppler.freedesktop.org/"
+COPYRIGHT="2008 Adam Batkin
+ 	Adobe Systems Inc.
+ 	2012-2015 Adam Reichold
+ 	2007-2017 Adrian Johnson
+ 	2008 Adrian Page
+ 	2013 Adrian Perez de Castro
+ 	2014 Aki Koskinen
+ 	2004-2017 Albert Astals Cid
+ 	2015 Aleksei Volkov
+ 	2006-2010 Alexander Neundorf
+ 	2012 Alexander Saprykin
+ 	2012 Alexey Pavlov
+ 	2016 Alok Anand
+ 	2015 André Esser
+ 	2015 André Guerreiro
+ 	2011 Andrea Canciani
+ 	2011 Andreas Hartmetz
+ 	2013 Anthony Granger
+ 	2012 Anthony Wesley
+ 	2014 Anuj Khare
+ 	2015-2016 Arseniy Lartsev
+ 	2012 Arseny Solokha
+ 	2015 Arthur Stavisky
+ 	2009, 2011 Axel Strübing
+ 	2014 Bogdan Cristea
+ 	2008 Boris Toloknov
+ 	2005-2008 Brad Hards
+ 	2010 Brian Cameron
+ 	2010 Brian Ewins
+ 	2016-2017 Caolán McNamara
+ 	2006-2016 Carlos Garcia Campos
+ 	2008 Carl Worth
+ 	2010 Christian Feuersänger
+ 	2009 Christian Persch
+ 	2008-2009 Chris Wilson
+ 	2017 Christoph Cullmann
+ 	2013 Christoph Duelli
+ 	1998-2003 Daniel Veillard
+ 	2011 Daiki Ueno
+ 	2005 Dan Sheridan
+ 	2011 Daniel Glöckner
+ 	2013 Daniel Kahn Gillmor
+ 	2011 Daniel Richard G.
+ 	2009 Darren Kenny
+ 	2009-2010 David Benjamin
+ 	2006 David Faure
+ 	1997, Derek B. Noonburg
```

Comment:
```diff
- 	1997, Derek B. Noonburg
+ 	1997 Derek B. Noonburg
```
BTW, if what you have right now in `COPYRIGHT` is the result of a grep then you are probably getting names from files such as `INSTALL` that mention authors which sometimes don't have much to do with the project itself.

In other words, if all these names come from source or header files, then that's fine.
If, however, they come from generic readme files or from generated files which might include names of people who maintain the tools used to generate them, then these names should probably be skipped.

Reply:
I see no point in including all of the authors and /or contributors.  In the early days of Haikuports we had a rule to crop it at 4 or 5 and perhaps add "et all" or to the end.
Since this is a fork of another project, I would credit the original project, the author of the new project and the maintain of the current  project and then and the others as a single group, including the year ranges.  Assume any year ranges prior to the fork belong to the original project.

Reply:
:+1:

BTW, I might have added more than 4 or 5 names in some recipes, but the contributor who most likes complete lists is... @alaviss :smile:

Reply:
I never liked `COPYRIGHT`. As far as I know, we and Debian-based distributions are the only ones who specify them. So if we wanted to include those, I would like to make sure that the list is correct, and complete.

With that said, I would not object if we add an guideline to keep the list short, it would greatly reduce the effort of playing with regexes and maintaining the list, which I would greatly appreciate :smile:.

Reply:
We should just follow what the upstream says for copyright. If it requires grepping through their files to find out, it's their problem and we should ask them to fix that.
I'm fine with using "1997-2018 Poppler contributors" or something similar, but we should check if the upstream is ok with that wording.